runtime/polkadot/src/lib.rs

```diff
@@ -835,7 +830,6 @@ pub enum ProxyType {
 	NonTransfer,
 	Governance,
 	Staking,
-	SudoBalances,
```

Comment:
I think we need to explicitly specify the codec index for this enum now? Otherwise `IdentityJudgement` variant may be broken.

Reply:
Yah fair enough, I'll introduce a dummy.

Reply:
Old transactions will be invalidated anyway through `CheckSpecVersion`.

And we probably don't have anything scheduled that uses the proxy feature?

Reply:
Ahh yeah.